config/mongoid.yml

```diff
@@ -9,6 +9,8 @@ common: &default_client
     timeout: <%= ENV['MONGOID_TIMEOUT'] || 0.5 %>
     ssl: <%= ENV['MONGOID_USE_SSL'] || false %>
     ssl_verify: <%= ENV['MONGOID_SSL_VERIFY'] || true %>
+    # not setting this env variable should result in backwards compatiblity
```

Comment:
I don't think we need this comment. This should have been there in the first place. :man_shrugging: 

```suggestion
```